plugins/modules/ec2_import_image_info.py

```diff
@@ -190,17 +175,18 @@ def main():
     )
 
     module = AnsibleAWSModule(argument_spec=argument_spec, supports_check_mode=True)
-    client = module.client("ec2", retry_decorator=AWSRetry.jittered_backoff())
+    client = module.client("ec2")
     params = {}
 
     if module.params.get("filters"):
         params["Filters"] = module.params["filters"]
     if module.params.get("import_task_ids"):
         params["ImportTaskIds"] = module.params["import_task_ids"]
 
-    import_image_info = helper_describe_import_image_tasks(client, module, **params)
```

Comment:
Can this `helper_describe_import_image_tasks()` function also be deleted from module_utils/ec2.py?